### PR TITLE
cavif: add page

### PR DESCRIPTION
**Pull request by Rudloff**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 1.5.3

### CODE REVIEW

File: pages/common/cavif.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# cavif
+
+> PNG/JPEG to AVIF converter.
+> More information: <https://github.com/kornelski/cavif-rs>.
+
+- Convert a JPEG file to AVIF:
+
+`cavif {{path/to/filename.jpg}}`
```
Comment (kbdharun) [inline pages/common/cavif.md:8]:
```suggestion
`cavif {{path/to/file.jpg}}`
```

File: pages/common/cavif.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Set the destination filename explicitly:
+
+`cavif {{path/to/filename.jpg}} --output {{path/to/filename.avif}}`
+
+- Overwrite the destination file if it already exists:
+
+`cavif --overwrite {{path/to/filename.jpg}}`
```
Comment (kbdharun) [inline pages/common/cavif.md:16]:
```suggestion
`cavif --overwrite {{path/to/file.jpg}}`
```

File: pages/common/cavif.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+> PNG/JPEG to AVIF converter.
+> More information: <https://github.com/kornelski/cavif-rs>.
+
+- Convert a JPEG file to AVIF:
+
+`cavif {{path/to/filename.jpg}}`
+
+- Set the destination filename explicitly:
+
+`cavif {{path/to/filename.jpg}} --output {{path/to/filename.avif}}`
```
Comment (kbdharun) [inline pages/common/cavif.md:12]:
```suggestion
- Set the output location explicitly:

`cavif {{path/to/file.jpg}} --output {{path/to/file.avif}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

> We suggest using `file` over `filename` in the path placeholder across our pages, I think the `file` placeholder would better fit this page. (Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths)

File: pages/common/cavif.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# cavif
+
+> PNG/JPEG to AVIF converter.
+> More information: <https://github.com/kornelski/cavif-rs>.
+
+- Convert a JPEG file to AVIF:
+
+`cavif {{path/to/file.jpg}}`
+
```
Comment (kbdharun) [inline pages/common/cavif.md:9]:
```suggestion

- Adjust the image quality (1-100) and convert a PNG file to AVIF:

`cavif --quality {{60}} {{path/to/file.png}}`

```
Maybe we could highlight an option for PNG too.

**Review by kbdharun — APPROVED** [review]

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**